cmd/memorydiskserver/memorydiskserver.go

```diff
@@ -63,6 +75,15 @@ func setHandler(w http.ResponseWriter, r *http.Request) {
 //}
 
 func main() {
+	// Read the starting value
+	dat, err := ioutil.ReadFile(STORE)
```

Comment:
Actually I forgot it truncates the file, which isn't what you want. Ok to leave it as it is then